dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -1400,10 +1400,11 @@ public String toString() {
             }
         }
         try {
-            forEach((k, v) -> { sb.append(k).append("=").append(v).append(", "); });
+            super.forEach((k, v) -> { sb.append(k).append("=").append(v).append(", "); });
         } catch (ConcurrentModificationException ex) {
             sb.append(" ConcurrentModificationException caught!");
         }
+        sb.setLength(sb.length()-2); //remove trailing comma and space
```

Comment:
If it there are no properties, then the StringBuffer will only have a { character in it.  As such you would set the length to -1 which will give you a IndexOutOfBoundsException.